### DISCUSSION TITLE
https://www.mickeymarse.dev/blog/mentor-crush/

### DISCUSSION THREAD

**Discussion by giscus** [mickeymarse/blogfolio#13] [Blog Posts Comments]
# https://www.mickeymarse.dev/blog/mentor-crush/

Website/portfolio/blog of Mickey Serrapica

https://www.mickeymarse.dev/blog/mentor-crush/

**Comment by shawnanuptra** [comment]
This resonated with me! Went to a conference the first time a couple of days ago, am writing a blog about it now.

**Reply by ckiyme** [reply]
Nice!
Thank you for reading. And, please, feel free to share your blog post. I'm curious now.

**Comment by shawnanuptra** [comment]
I go through the experience of my first ever conference in this blog! Feel
free to message me anytime :]
https://www.shawn-am.co.uk/blog/my-first-conference

**Reply by ckiyme** [reply]
That sounds like an amazing experience! There were so many that I follow at that conference. I really feel bad for missing it. 🥲 
But I'm glad you loved it and you felt inspired. Also, congrats on your new job! Way to go! 😃

**Comment by shawnanuptra** [comment]
Thank you so much hehe yeah it was goated. I'm sure there are other times
you get to attend!

**Reply by ckiyme** [reply]
I was at Peckham Digital last weekend, which was also so stunning!
I want to go to more of these mind-blowing conferences!!!
